creator-node/src/components/healthCheck/healthCheckComponentService.js

```diff
@@ -87,7 +95,13 @@ const healthCheckVerbose = async ({ libs } = {}, logger, sequelize, getMonitors,
     MONITORS.MAX_FILE_DESCRIPTORS,
     MONITORS.ALLOCATED_FILE_DESCRIPTORS,
     MONITORS.RECEIVED_BYTES_PER_SEC,
-    MONITORS.TRANSFERRED_BYTES_PER_SEC
+    MONITORS.TRANSFERRED_BYTES_PER_SEC,
+    MONITORS.THIRTY_DAY_ROLLING_SYNC_SUCCESS_COUNT,
```

Comment:
ignore these -- once https://github.com/AudiusProject/audius-protocol/pull/1537 is merged, these changes will not be shown in the diff

Reply:
fyi can get rid of this diff by opening the PR against `vg-expose-sync-history-health-check` instead of master, and once thats merged just repoint it back to master